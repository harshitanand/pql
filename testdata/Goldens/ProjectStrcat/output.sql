SELECT "EventType" || ' in ' || "State" AS "Description" FROM "StormEvents";
