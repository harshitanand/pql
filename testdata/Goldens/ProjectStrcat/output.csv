Description
Waterspout in ATLANTIC SOUTH
Heavy Rain in FLORIDA
Tornado in FLORIDA
Thunderstorm Wind in GEORGIA
Thunderstorm Wind in MISSISSIPPI
